tracer/src/Datadog.Trace/Debugger/Upload/SnapshotUploadApi.cs

```diff
@@ -49,10 +49,12 @@ public override async Task<bool> SendBatchAsync(ArraySegment<byte> data)
 
             var request = _apiRequestFactory.Create(new Uri(uri));
 
+            Log.Debug("SnapshotUploadApi: Sending snapshots to {Uri}", uri);
```

Comment:
Are you using the same endpoint as the debugger? It may not matter, but from the next version of agent\tracer we have a new endpoint for snapshots. https://github.com/DataDog/dd-trace-dotnet/pull/7388